config/routes.rb

```diff
@@ -86,6 +87,7 @@
         get 'condition_form', :path => 'condition-form'
       end
     end
+    resources :qing_groups, except: :new
```

Comment:
Can we add `:path => 'qing-groups'` for consistency here? Not that the user will ever see this URL in the addy bar...